db/columns/operations/select.py

```diff
@@ -84,43 +84,35 @@ def get_column_name_from_attnum(table_oid, attnum, engine, connection_to_use=Non
 def get_column_default_dict(table_oid, column_index, engine, connection_to_use=None):
     table = reflect_table_from_oid(table_oid, engine, connection_to_use)
     column = table.columns[column_index]
-    default_dict = None
-    if column.server_default is not None:
-        default_dict = {
-            "server_default": column.server_default,
-            "is_dynamic": _is_default_expr_dynamic(column.server_default),
-            "sql_text": str(column.server_default.arg)
-        }
-    else:
+    if column.server_default is None:
         return
-    if default_dict.get("is_dynamic"):
+
+    is_dynamic = _is_default_expr_dynamic(column.server_default)
+    sql_text = str(column.server_default.arg)
+
+    if is_dynamic:
         warnings.warn(
             "Dynamic column defaults are read only", DynamicDefaultWarning
         )
+        default_value = sql_text
     else:
         # Defaults are stored as text with SQL casts appended
         # Ex: "'test default string'::character varying" or "'2020-01-01'::date"
         # Here, we execute the cast to get the proper python value
-        default_dict.update(
-            executed_constant=execute_statement(
-                engine,
-                select(text(default_dict['sql_text'])),
-                connection_to_use
-            ).first()[0]
-        )
-    return default_dict
+        default_value = execute_statement(
+            engine,
+            select(text(sql_text)),
```

Comment:
Is it possible to set a `sql command` as the default char value with the intention of causing a SQL injection? Since a cast accompanies it, I am assuming it gets converted into a value, but not sure if this can be bypassed

Reply:
In this specific instance, I think we're safe since the default-setting mechanism seems to sanitize input (wrapping in quotes and casting).  However, I'm not 100% sure.  I _definitely_ think we need to figure out some automated audit for SQL injection problems with our app.

Reply:
On a side note:  I consider this a beta-or-later problem, since our initial users should be considered "friendly", i.e., they own the DB they're running queries against.

Reply:
Can we create an issue to track potential SQL injection issues?

Reply:
> Can we create an issue to track potential SQL injection issues?

#958